tcms/testplans/static/testplans/js/get.js

```diff
@@ -1,4 +1,5 @@
-var expandedTestCaseIds = [];
+var expandedTestCaseIds = [],
+    fadeAnimationTime = 500;
```

Comment:
this seems to be old, shouldn't this PR be using the animate function ?

Reply:
Yeah, but I noticed that with removing with introduce a bug, when we remove a testcase from a plan we fadeout the row and then remove it from the dom, no fadein is used(animate also will not do the trick).